w01-controlling-the-dom/d4-dawn-intro-jquery/README.md

```diff
@@ -0,0 +1,190 @@
+var el = document.getElementsByTagName("h1");
+// jQuery
+var el = $('h1');
+```
+
+#### Add Dynamic Changes to Events with Functions
+Add a function
+```javascript
+// Javascript
+el.addEventListener("click", function() {
+  alert("you clicked the element this listener is attached to");
+});
+
+// jQuery
+el.on('click', function() {
+  alert("Your click triggered this listener!");
+})
+
+// Other Event Listeners
+// "mouseenter"
+// "mouseleave"
+// "submit"
+```
+
+Change or add a style attribute value
+```javascript
+// Javascript
+var arr = document.getElementsByClassName('text-good');
+// console.log(arr)
+
+for(i = 0; i < arr.length; i++) {
+    console.log(i)
+    console.log(arr[i]);
+    arr[i].style.color = "green"
+    // arr[i].style.display= "none" // hide the element
+}
+
+// jQuery
+var arr = $('text-good');
+arr.each(function(index) {
+  $(this).text("number " + index);
+  $(this).css("color", "blue");
+})
+```
+
+Change text
+```javascript
+// Javascript
+el.innerText = "New Text!"
+
+// jQuery
+el.text("New text!")
+```
+
+Add class
+```javascript
+// Javascript
+el.classList.add("danger");
+
+// jQuery
+el.addClass("danger");
+```
+
+Prevent Default Behavior
+```javascript
+// Javascript
+var button = document.querySelector("a#san-francisco_cta");
+button.onclick = function(e){
+    event.preventDefault(); // SUPER IMPORTANT PART
+    alert("Hahah! Now you get me instead")
+};
+
+// jQuery
+var button = $('a#san-francisco_cta');
+button.on('click', function(e) {
+    event.preventDefault(); // SUPER IMPORTANT PART
+    console.log("if you see this message in your Dev Tools console, then your prevent default is working");
+})
+```
+
+### Demo (set up new project and adding jQuery CDN)
+
+1. Make a new directory: ```mkdir intro-jquery```
+2. Change directories to the one you just made: ```cd intro-jquery```
+3. Create three files: ```index.html```, ```styles.css```, and ```scripts.js```. Use the ```touch``` command followed by the file name, like this: ```touch index.html```.
+4. Add the jQuery CDN link to the ```<head>``` tag in your ```index.html``` file. Use a script tag to include it like this:
+```<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js">```
+
+  It should look like this:
+
+    ```html
+    <html>
+    <head>
+      <title>Your Page</title>      
+      <!-- BOOTSTRAP -->
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+
+      <!-- LOCAL CSS -->
+      <link rel="stylesheet" href="styles.css">
+
+      <!-- REMOTE SCRIPT (FROM CONTENT DELIVERY NETWORK (CDN)) -->
+      <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+
+      <!-- LOCAL SCRIPTS -->
+      <script src="scripts.js"></script>
+    </head>
+    <body>
+      <!-- YOUR HTML with CSS classes and ids -->
+    </body>
+    </html>
+    ```
+1. Add this code to your `scripts.js`:
+  ```js
+  $(document).ready(function(){
```

Comment:
I think this is the first time they've seen `$(document).ready` does it warrant more discussion?